.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "tabWidth": 4,
+  "trailingComma": "es5",
+  "singleQuote": true
+}
```

Comment:
Isn't that already in the project? I remember @jacobhallenborgprototyp already added prettier. In fact there's already a `prettierrc.json` file in the project.